packages/web-components/src/components/va-number-input/va-number-input.tsx

```diff
@@ -0,0 +1,150 @@
+import {
+  Component,
+  Element,
+  Prop,
+  Host,
+  h,
+  Event,
+  EventEmitter
+} from '@stencil/core';
+
+@Component({
+  tag: 'va-number-input',
+  styleUrl: 'va-number-input.css',
```

Comment:
Could we point to the CSS file for the text input? Alternatively, could we `@import` that file in `va-number-input.css` so we don't have to redefine a lot of the same things?

Reply:
I tried to point it to the va-text-input.css but that gave a Rollup error so I chose to do an import in css file instead which worked correctly